indexers/src/handlers/transactions/handlePostRequest.transaction.handlers.ts

```diff
@@ -0,0 +1,29 @@
+import assert from "assert";
+import { SupportedChain } from "../../types";
+import { HandlePostRequestsTransaction } from "../../types/abi-interfaces/HandlerV1Abi";
+import { RelayerService } from "../../services/relayer.service";
+
+/**
+ * Handles the handlePostRequest transaction from Hyperbridge
+ */
+async function handlePostRequestTransaction(
+  transaction: HandlePostRequestsTransaction,
+  chain: SupportedChain,
+): Promise<void> {
+  assert(
+    transaction.args,
+    "No handlePostRequestTransaction args found in ${network} network",
+  );
+
+  await RelayerService.handlePostRequestTransaction(transaction, chain);
+}
+
+// Handle the handlePostRequest transaction for the Ethereum Sepolia chain
+export async function handleEthereumSepoliaPostRequestTransactionHandler(
+  transaction: HandlePostRequestsTransaction,
+): Promise<void> {
+  await handlePostRequestTransaction(
+    transaction,
+    SupportedChain.ETHEREUM_SEPOLIA,
```

Comment:
Why is there a special function for sepolia?  we can get the chainId from the transaction.

Reply:
This has been updated to get the chainId from the transaction